src/core/app.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { BrowserRouter as Router, Route, Switch, Redirect } from 'react-router-dom';
+import { ApolloProvider } from "react-apollo";
+
+import { AddTool, EditTool, Tools } from '../features';
+
+function App({ store, client }){
+  return(
+    <ApolloProvider store={store} client={client}>
+      <Router>
+          <React.Fragment>
```

Comment:
no need of `Fragment` here - here you have only one child node (`Switch`)

Reply:
Done